docs/getting-started/nas/synology.md

```diff
@@ -91,10 +91,135 @@ This guide describes how to set up PhotoPrism using the new Synology user interf
 ### First Steps
 
 Our [First Steps 👣](../../user-guide/first-steps.md) tutorial guides you through the user interface and settings to ensure your library is indexed according to your individual preferences.
+-->
 
+## Setup using Docker and MariaDB
 
+### Prerequisites
+- Docker is installed (called 'Container Manager' to Synology.)
+- Create some folders for Photoprism to store configuration data and photos.
+  
+  ![Photoprism_1](./img/synology/Photoprism_1.jpg){ class="shadow" }
 
+- For testing purposes, add some pictures to your photos folder.
+- Later, if you're okay with your setup, you can link your pictures to the photos folder.
+
+### Preparing MariaDB
+- Launch `Container Manager` to access Docker
```

Comment:
Maybe container manage isn't "Docker". So just omit "to access Docker". Still makes sense.